Services/UserService/Validators/CreateUserRequestValidator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DTO;
+using DTO.BrokerRequests;
+using DTO.RestRequests;
+using FluentValidation;
+
+namespace UserService.Validators
+{
+    public class CreateUserRequestValidator : AbstractValidator<CreateUserRequest>
+    {
+        public CreateUserRequestValidator()
+        {
+            RuleFor(user => user.Email)
+                .NotEmpty()
+                .WithMessage(ErrorsMessages.EmailEmpty)
+                .MaximumLength(50)
```

Comment:
Not have a NullCheck?

Reply:
No need if NotEmpty() it is also checked for NotNull()